docs/marketplace/dev/feature-integration-guides/marketplace-product-offer-checkout-feature-integration.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Marketplace Product Offer + Checkout feature integration
+last_updated: Apr 28, 2021
+description: This document describes the process how to integrate the Marketplace Product Offer + Checkout feature into a Spryker project.
+---
+
+This document describes how to integrate the [Marketplace Product Offer + Checkout feature integration]() feature into a Spryker project.
+
+## Install feature core
+Follow the steps below to install the Marketplace Product Offer + Checkout feature integration feature core.
+
+### Prerequisites
+To start feature integration, integrate the required features:
+
+| NAME | VERSION | INTEGRATION GUIDE |
+|-|-|-|
+| Spryker Core | master | [Spryker Core feature integration](https://documentation.spryker.com/docs/spryker-core-feature-integration)  |
```

Comment:
Checkout feature should be mentioned as well.

Reply:
Is `202001.0` the correct version? Copied it from https://github.com/spryker/spryker.github.io/blob/master/docs/marketplace/dev/feature-integration-guides/marketplace-shipment-checkout-feature-integration.md